packages/@react-spectrum/table/src/table.css

```diff
@@ -14,6 +14,7 @@
   user-select: none;
   display: flex;
   flex-direction: column;
+  min-height: 0;
```

Comment:
What does this fix (what specific configuration of a Flex w/ a TableView within it)?

Reply:
nvm, Rob updated me on the example